disease/query.py

```diff
@@ -317,40 +306,59 @@ def _add_merged_meta(self, response: Dict) -> Dict:
+        other_ids = vod.get('xrefs', None)
+        if other_ids:
+            ids += other_ids
+        for concept_id in ids:
             prefix = concept_id.split(':')[0]
             src_name = PREFIX_LOOKUP[prefix.lower()]
             if src_name not in sources_meta:
                 sources_meta[src_name] = self._fetch_meta(src_name)
         response['meta_'] = sources_meta
         return response
 
-    def _format_merged_record(self, merged_record: Dict) -> Dict:
-        """Transform merged record (as served by DB) into response object
-        format.
-        :param Dict merged_record: merged record as returned by DB call
-        :return: formatted record
-        """
-        del merged_record['label_and_type']
-        merged_record['concept_ids'] = merged_record['concept_id'].split('|')
-        del merged_record['concept_id']
-        return merged_record
-
-    def _add_merged_record(self, response: Dict, merge_ref: str) -> Dict:
-        """Add referenced concept ID group to response object.
-        :param Dict response: in-progress response object. Should have
-            match_type attribute filled in already.
-        :param str merge_ref: primary key for concept ID group lookup.
-        :return: completed response object.
+    def _add_vod(self, response: Dict, record: Dict, query: str,
+                 match_type: MatchType) -> Dict:
+        """Format received DB record as VOD and update response object.
+        :param Dict response: in-progress response object
+        :param Dict record: record as stored in DB
+        :param str query: query string from user request
+        :param MatchType match_type: type of match achieved
+        :return: completed response object ready to return to user
         """
-        merged_record = self.db.get_record_by_id(merge_ref, False, True)
-        if not merged_record:
-            logger.warning(f"Could not retrieve merged record for concept "
-                           f"ID group {merge_ref} "
-                           f"by way of query {response['query']}")
-            response['match_type'] = MatchType.NO_MATCH
-            return response
-        response['record'] = self._format_merged_record(merged_record)
+        vod = {
+            'id': f'normalize:{query}',
+            'type': 'DiseaseDescriptor',
+            'value': {
+                'type': 'Disease',
+                'disease_id': record['concept_id']
+            },
+            'label': record['label'],
+            'extensions': [],
+        }
+        if 'other_ids' in record:
+            vod['xrefs'] = record['other_ids']
+        if 'aliases' in record:
+            vod['alternate_labels'] = record['aliases']
+        if 'pediatric_disease' in record and \
+                record['pediatric_disease'] is not None:
+            vod['extensions'].append({
+                'type': 'Extension',
+                'name': 'pediatric_disease',
+                'value': record['pediatric_disease']
```

Comment:
The `value` for an `Extension` should be a list https://vrsatile.readthedocs.io/en/latest/value_object_descriptor/vod_index.html#extension

Reply:
We should also use the `Extension` class from `schemas.py` here as well.

Reply:
It sounds like that's a typo in the docs under `Extension.value` -- Alex says it can be any primitive value/structured object (incl. but not necessarily an array)

Reply:
Ah okay! I will definitely have to fix this for variant normalization. Thanks for letting me know!